File: medis-ui/src/app/main/patients/classes/address.ts
export class Address {
  constructor(
    public region?: String,
    public city?: string,
    public address_type?: string,
    public address_name?: string,
    public phone_number?: string
  ) {
    this.region = region ? region : '';
    this.city = city ? city : '';
    this.address_type = address_type ? address_type : '';
    this.address_name = address_name ? address_name : '';
    this.phone_number = phone_number ? phone_number : '';
  }
}

File: medis-service/src/main/java/io/mersys/medis/documents/patient/Address.java

package io.mersys.medis.documents.patient;

import java.io.Serializable;

import javax.validation.constraints.NotBlank;

import org.springframework.data.annotation.TypeAlias;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
@TypeAlias("Address")
@Data
@AllArgsConstructor
@NoArgsConstructor
public class Address{
	
	@NotBlank
	private String region;
	
	@NotBlank
	private String city;
	
	private String addressType;
	
	private String address;
	
	private String phoneNumber;

}
